src/swirrl/auth0/client.clj

```diff
@@ -44,7 +46,18 @@
                              :client-id (:client-id auth0)
                              :client-secret (:client-secret auth0)
                              :audience (:aud auth0)})
-      (:body)))
+      (:body)
+      (vary-meta assoc :timestamp (time/now))))
```

Comment:
Is there a reason to store this as metadata on the response body, vs associng a possibly namespaced key into the response body?  Equality?

Reply:
Mainly because I don't want to accidentally break any users. And it is actual meta-data.